R/get_source_expressions.R

```diff
@@ -152,6 +152,19 @@ lint_parse_error <- function(e, source_expression) {
   lint_parse_error_nonstandard(e, source_expression)
 }
 
+fixup_line <- function(line) {
+  # helper to ensure a valid string is provided as line where necessary handles tow cases:
```

Comment:
i'm not understanding this comment, some typos?

> handles tow cases

Reply:
Yeah, should be "two cases"
The problem is some situations can wind up with `line` being either `NA_character_` or containing an invalid multibyte character. Both of these will mess up the `Lint()` object, so need to be addressed.